### ISSUE TITLE
Keyboard navigation is broken when grouping is used

### ISSUE THREAD

**Issue by zoechi** [bwu-dart/bwu_datagrid#91]
Only data lines can be navigated using the keyboard, once trying navigating out of a current group content lines and keyboard navigation doesn't work anymore until the focus is restored using the mouse.
